### PR TITLE
Changes in css/hover

### PR DESCRIPTION
**Pull request by MirekSobota**
In this update, I have:

- Added or changed some properties in CSS to fix tile display on small resolutions.
- Added hover effects to the posters in Movie and People lists.
- Moved the image folder into the TileMovie folder.

Let me know what you think!"

### CODE REVIEW

**Review by PawelOwiesek — APPROVED** [review]
It looks very good and finally the first page is responsive, good job.

File: src/common/TileList/TileMovie/styled.js
Comment (DorotaZielinska) [inline src/common/TileList/TileMovie/styled.js:1]: thanks for finding bug with width in MovieTileList

File: src/common/TileList/TileMovie/Image/styled.js
Comment (DorotaZielinska) [inline src/common/TileList/TileMovie/Image/styled.js:1]: That looks for better solution with overflow, cursor and inset than before

Comment (MirekSobota) [reply src/common/TileList/TileMovie/Image/styled.js:1]: Happy to hear it! Thanks

**Pull request merged**